src/api/Gemfile

```diff
@@ -19,7 +19,7 @@ gem 'thinking-sphinx', '> 3.1'
 # to paginate search results
 gem 'kaminari'
 # as abstract HTML of the bratwurst 'theme'
-gem 'haml'
+gem 'haml-rails'
```

Comment:
What is the advantage of changing haml by haml-rails? You wrote this in the commit message:

_To be more in sync with current rails releases, this commit now requires haml-rails instead of haml._

But I am not sure what you mean :confused: